edxval/tests/test_api.py

```diff
@@ -1918,12 +1938,10 @@ def test_video_transcript_deletion(self):
 
         # This will replace the transcript for an existing video and delete the existing transcript
         new_transcript_url = api.create_or_update_video_transcript(
-            self.video_id,
-            'ur',
-            'overwatch.srt',
-            TranscriptFormat.SRT,
-            TranscriptProviderType.CIELO24,
-            ContentFile(FILE_DATA)
+            video_id=self.video_id,
+            language_code='ur',
+            metadata=dict(provider=TranscriptProviderType.CIELO24),
```

Comment:
don't we need to pass the`file_format` in the metadata?

Reply:
Don't need to update as its already in SRT format.